src/qtism/data/storage/xml/marshalling/OutcomeDeclarationMarshaller.php

```diff
@@ -96,8 +96,8 @@ protected function marshall(QtiComponent $component)
             $element->appendChild($lookupTableMarshaller->marshall($component->geTLookupTable()));
         }
 
-        if ($component->getExternalScored() !== null) {
-            static::setDOMElementAttribute($element, 'externalScored', ExternalScored::getNameByConstant($component->getExternalScored()));
+        if (Version::compare($version, '2.2.0', '>=') === true && $component->isExternallyScored()) {
```

Comment:
Is this method existing in `QtiComponent`?

Reply:
No, nor are the lookupTable ones or previous. They're declared in the OutcomeDeclaration object child of QtiComponent, and $componenet is supposed to be an instance of OutcomeDeclaration here, I guess.